docs/team/transfer-apps.md

```diff
@@ -0,0 +1,30 @@
+title: Sematext Transfer Apps
+description: Inviting team members to your account means they get access to all your Apps, dashboards, notification hooks, alert rules, and everything else!
+
+No level of Access can change the Owner status of an App. To transfer the 
```

Comment:
The first sentence sounds weird.  An easier version:
Only App Owner can change an App ownership by transferring that App to another user.